chat-plugins/modlog.js

```diff
@@ -168,13 +218,13 @@ async function runModlog(rooms, searchString, exactSearch, maxLines) {
 	fileNameList = fileNameList.map(filename => `${LOG_PATH}${filename}`);
 
 	let regexString;
+	let searchType = 0;
 	if (!searchString) {
 		regexString = '.';
```

Comment:
I think it would be neater if this case were moved to ``formatRegexString()``

Reply:
Then, if you find it convenient, you could do set the variables using destructuring.

```js
const {searchRegex: regexString, type: searchType} = formatRegexString(searchString);
```

Reply:
I havent tested this, but wouldnt the scope of the const be inside the``else { }`` part?

Reply:
> I think it would be neater if this case were moved to formatRegexString()
  
The idea is that then there would be no ``if-else`` statement here :)